terminus-ui/input/testing/src/test-components.ts

```diff
@@ -31,24 +31,18 @@ import {
 export class SimpleFormControl {
   control: FormControl | undefined = new FormControl();
 
-  @ViewChild(TsInputComponent)
+  @ViewChild(TsInputComponent, {static: true})
   inputComponent: TsInputComponent;
 }
 
 
-@Component({
-  template: `
-    <ts-input
-      [datepicker]="true"
-    ></ts-input>
-  `,
-})
+@Component({template: `<ts-input [datepicker]="true"></ts-input>`})
```

Comment:
honestly i'd prefer the old layout, but might not go with your rule well...

Reply:
I'd never do this in a component, but when dealing with test components, if it's a single line of template I don't mind the compact nature. Open to enforcing it across test files too if our team wants 👍